terraform/main.tf

```diff
@@ -0,0 +1,17 @@
+terraform {
+  required_providers {
+    azurerm = {
+      source  = "hashicorp/azurerm"
+      version = "~> 3.7.0"
+    }
+  }
+}
+
+# Configure the Microsoft Azure Provider
+provider "azurerm" {
+  features {}
+}
+
+data "azurerm_resource_group" "benefits" {
+  name = "RG-CDT-PUB-VIP-CALITP-D-001"
```

Comment:
This seems like something that should be in source control?

Reply:
It's not sensitive, if that's what you mean.

If you are referring to hard-coding the value, we could parameterize it / make it a variable, but no benefit in doing so right now.